compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processor/MangledNamesProcessor.kt

```diff
@@ -60,17 +60,17 @@ class MangledNamesProcessor : AbstractTestProcessor() {
         override fun visitFunctionDeclaration(function: KSFunctionDeclaration, data: MutableMap<String, String>) {
             if (function.simpleName.asString() in IGNORED_FUNCTIONS) return
             super.visitFunctionDeclaration(function, data)
-            data[function.simpleName.asString()] = resolver.getJvmName(function)
+            data[function.simpleName.asString()] = resolver.getJvmName(function)!!
```

Comment:
might be better to just change the map to accept null values to match the method signature.